internal/query/search_query.go

```diff
@@ -263,6 +263,70 @@ func NumberComparisonFromMethod(m domain.SearchMethod) NumberComparison {
 	}
 }
 
+func NewInQuery(c Column, value string, compare TextComparison, inColumn, selectedColumn Column, linkingEquals []string) (*InQuery, error) {
```

Comment:
there's already a `ListQuery`... i know it would take same refactoring but IMHO it might make sense to do the extra effort
and regarding this `NewInQuery`:
- it must not be limited to Text columns
- linkingEquals must not be a `[]string` why don't we just pass the columns and build the links where needed
this would also ensure that there's no sql injection possible

Reply:
NewInQuery included into ListQuery with possibility to use a SubQuery or Slice of data as datasource